src/main/java/Deadline.java

```diff
@@ -6,9 +6,14 @@ public Deadline(String description, String by) {
         this.by = by;
     }
 
+    public Deadline(String description, boolean done, String by) {
```

Comment:
Perhaps consider renaming `done` to `isDone` as per the coding style. Boolean field names should start with a verb.